<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="myWidget" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>801</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color (R G B)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Delete Object</string>
     </property>
    </widget>
    <widget class="QComboBox" name="MeshBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="LineBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Line Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Current Object</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BoundBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Bound Box</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-30</y>
       <width>1231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="HideBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Show Object</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ColorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Change Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Reset Position</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="RegularDivide">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Regular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="IrregularDivide">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Irregular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UniformSmooth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Uniform</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CordSmooth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Cord</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mesh Subdivide</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Surface Smooth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MeanCurvSmooth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Mean Curv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MeanValSmooth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Mean Val</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBoardBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Check Board</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heat Diffusion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExplicitDiffusion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Explicit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExplicitDiffusion_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Implicit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExplicitDiffusion_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Gaussian</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Curvature Calculation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GaussianCurvature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Gaussian</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PrincipalCurvature">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Principal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PrincipalSmooth">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Smooth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CalcStreamLine">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Stream</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SphericalParameterize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>SphPara</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Mesh">
     <property name="title">
      <string>Open Mesh</string>
     </property>
     <addaction name="actionOpen_obj_file"/>
     <addaction name="actionOpen_ply_file"/>
    </widget>
    <addaction name="menuOpen_Mesh"/>
   </widget>
   <widget class="QMenu" name="menuSurface">
    <property name="title">
     <string>Surface</string>
    </property>
    <addaction name="actionGenerate_a_Surface"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSurface"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_obj_file">
   <property name="text">
    <string>Open .obj file</string>
   </property>
  </action>
  <action name="actionOpen_ply_file">
   <property name="text">
    <string>Open .ply file</string>
   </property>
  </action>
  <action name="actionGenerate_a_Surface">
   <property name="text">
    <string>Generate a Surface</string>
   </property>
  </action>
  <action name="actionSurface_Smoothing">
   <property name="text">
    <string>Surface Smoothing</string>
   </property>
  </action>
  <action name="actionRegular_Subdivide">
   <property name="text">
    <string>Regular Subdivide</string>
   </property>
  </action>
  <action name="actionIrregular_Subdivide">
   <property name="text">
    <string>Irregular Subdivide</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
